ReactiveUI/Cocoa/CommonReactiveSource.cs

```diff
@@ -278,30 +234,74 @@ void resetupAll(IReadOnlyList<TSectionInfo> newSectionInfo)
             //
             // TODO: Instead of listening to Changed events and then reseting,
             // we could listen to more specific events and avoid some reloads.
-            disp.Add(sectionChanging.Subscribe(_ => {
+            disp.Add(sectionChanging.ObserveOn(ImmediateScheduler.Instance).Subscribe(_ => {
```

Comment:
Several refreshes and restarts later, I can build again.

Have pushed an update to remove explicit immediate scheduler usage. Also fixed a minor type issue on non-UNIFIED builds, even though it had built fine until now. Seriously, Xamarin, I can't even.